lib/ts/recipe/session/recipeImplementation.ts

```diff
@@ -238,8 +237,7 @@ export default function getRecipeInterface(
                 return session;
             } catch (err) {
                 if (err.type === STError.UNAUTHORISED) {
-                    logDebugMessage("getSession: Clearing cookies because of UNAUTHORISED response");
-                    clearSessionFromCookie(config, res);
+                    logDebugMessage("getSession: Cookies will get cleared because of UNAUTHORISED response");
```

Comment:
move this debug log to the error handler place and can remove this whole try catch.